src/components/Notifications.tsx

```diff
@@ -1,41 +1,64 @@
 import { removeNotification, useNotifications } from "@src/hooks/notifications";
 import type { Notifications } from "@src/types";
 import React from "react";
-import { AlertList } from "react-bs-notifier";
+import Toast from "react-bootstrap/Toast";
 
 function NotificationDetails({ details }: { details: string[] }) {
   if (details.length === 0) {
     return null;
   }
   if (details.length === 1) {
-    return <span>{details[0]}</span>;
+    return (
+      <Toast.Body>
+        <span>{details[0]}</span>
+      </Toast.Body>
+    );
   }
   return (
-    <ul>
-      {details.map((detail, index) => {
-        return <li key={index}>{detail}</li>;
-      })}
-    </ul>
+    <Toast.Body>
+      <ul>
+        {details.map((detail, index) => {
+          return <li key={index}>{detail}</li>;
+        })}
+      </ul>
+    </Toast.Body>
   );
 }
 
 export default function Notifications() {
   const notifications = useNotifications();
+
   const alerts = notifications.map(
-    ({ message: headline, details = [], ...attrs }, i) => {
-      const message = <NotificationDetails details={details} />;
-      return { id: i, headline, message, ...attrs };
+    ({ message: headline, details = [], timeout, type }, i) => {
+      console.log(details);
+      return (
+        <Toast
+          className={`bg-${type}`}
+          key={`notification-${i}`}
+          autohide={timeout > 0}
+          delay={timeout > 0 ? timeout : undefined}
```

Comment:
There could be a bug with this, not blocking, but if I click 6 times on the copy header in the top bar. Only 3 notifications disappear automatically